lang/LangPrimSource/PyrSched.cpp

```diff
@@ -234,14 +234,17 @@ using monotonic_clock = std::conditional<std::chrono::high_resolution_clock::is_
 static std::chrono::high_resolution_clock::time_point hrTimeOfInitialization;
 
 #ifdef SC_ABLETON_LINK
-	void initLink();
+  class LinkClock {
+    public:
+      static void Init();
+  };
```

Comment:
Forward declaration becomes more verbose because there is no header file and the entire class + static method needs to be defined in `PyrSched.cpp`.